tubearchivist/docker-compose.yml

```
@@ -0,0 +1,48 @@
+version: '3.7'
+services:
+  app_proxy:
+    environment:
+      APP_HOST: tubearchivist_server_1
+      APP_PORT: 8000
+  server:
+    image: bbilly1/tubearchivist:v0.5.7@sha256:20ce26849d0bf00db9d85300f8205af24d4cd507dd7ab53b55d48a33544ce5b6
+    volumes:
+      - ${APP_DATA_DIR}/data/media:/youtube
+      - ${APP_DATA_DIR}/data/cache:/cache
+    environment:
+      - ES_URL=http://elasticsearch_db:9200
+      - REDIS_CON=redis://redis_db:6379
```

Go for the full service names and get rid of the `_db` in the service names.
```suggestion
      - ES_URL=http://tubearchivist_elasticsearch_1:9200
      - REDIS_CON=redis://tubearchivist_redis_1:6379
```